kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -360,4 +365,6 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
         TimeUnit.MILLISECONDS)
     }
   }
+
+  private[kyuubi] def isEngineContextStopped: Boolean = false
```

Comment:
why hard code this?

Reply:
oh, this is an abstract class located in the common module, returning `false` is fine